docs/federation.md

```diff
@@ -0,0 +1,116 @@
+# Federation
+
+## Introduction
+
+The DACCS Network (“Network”) is a federated network that consists of independently operated and maintained computing
+infrastructure (“Node”) that deploy the DACCS software stack and connect with one another under the framework laid out
+in this document. While anyone is free is deploy the software stack on their own hardware and use it as they choose, in
+order to join the Network as a Node, the Node needs to provide certain capabilities and minimum operating standards, as
+laid out in this document, and which must be agreed upon by the person(s) responsible for maintaining the Node (“Node
+Administrator”).
+
+## Principles of Federation
+
+The following sections describe the requirements for operating and maintaining a Node.
+
+### Operations
+
+- Deployment of software stack: Each Node should deploy the DACCS software stack. Node Administrators are required to
+  upgrade the software stack to the latest DACCS stack within two weeks of the updates release date.
+- Node maintenance: Node Administrators accept that joining the Network implies that they are responsible for
+  provisioning, funding and maintaining their own Node.
+- Node security: Node Administrators are responsible for maintaining secure access to their node and for coordinating
+  with
+  the hosting institution regarding any security related considerations prior to the deployment of the Node. Admin
+  access
+  to the Node must be restricted to listed Node Administrators.
+- Downtime policy: A Node should strive to maintain downtime of no more than 5%. If a Node needs to be taken offline for
```

Comment:
If we notify the users before hand, can we have a downtime of more than 5%?

Example when changing our production machine, we needed a downtime of 1 day. 

We sometime also have unscheduled power outage as well that lasts way more than just 1 hour.

Currently if an event is scheduled, we notify all our Ouranos Jupyter users.  In a federation, should we notify all users in the federation?

Reply:
Maybe we should clarify that this is unscheduled downtime. Scheduled downtime can of course be as long as needed as long as the users are warned beforehand (see next sentence). 

We should reword this to talk about having a policy to recover from unscheduled outages within X number of hours/days instead of in terms of a percentage